### PR TITLE
created elevator and wrist subsystems

### PR DESCRIPTION
**Pull request by AlenaNguyen**

### CODE REVIEW

File: src/main/java/frc/robot/RobotMap.java [hunk 1]
```diff
@@ -57,6 +57,15 @@ public static final class mapCollector {
     // public static final int PIVOT_ABSOLUTE_ENCODER_DIO = 1;
   }
 
+  public static final class mapElevator {
```
Comment (AngelaKimmy) [inline src/main/java/frc/robot/RobotMap.java:60]: Remember to bump the motors by 10 for each subsystem

File: src/main/java/frc/robot/subsystems/Elevator.java [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    position = MathUtil.clamp(position, prefElevator.elevatorMinPos.getValue(),
+        prefElevator.elevatorMaxPos.getValue());
+
+    leftMotor.set(ControlMode.Position, position);
+    rightMotor.set(ControlMode.Position, position);
+  }
+
+  public double getElevatorEncoderCounts() {
+    return rightMotor.getSelectedSensorPosition();
+  }
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Elevator.java:76]: We probably want to a getElevatorDistance that converts counts to inches (or meters)

Comment (TaylerUva) [reply src/main/java/frc/robot/subsystems/Elevator.java:76]: for robot preferences and things we should use distance so it's easier to read later

File: src/main/java/frc/robot/subsystems/Wrist.java [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+  public void setWristSpeed(double speed) {
+    wristMotor.set(ControlMode.PercentOutput, speed);
+  }
+
+  public void setWristPosition(double position) {
+    position = MathUtil.clamp(position, prefWrist.WristMinPos.getValue(),
+        prefWrist.WristMaxPos.getValue());
+
+    wristMotor.set(ControlMode.Position, position);
+  }
```
Comment (TaylerUva) [inline src/main/java/frc/robot/subsystems/Wrist.java]: We probably want to name this setWristAngle

**Review by TaylerUva — CHANGES_REQUESTED** [review]

**Review by TaylerUva — APPROVED** [review]

**Pull request merged**